test/systemtest/conftest.py

```diff
@@ -7,139 +7,43 @@
 import shutil
 import subprocess
 from pathlib import Path
+import locale
```

Comment:
I guess we don't run flake8 on these files? But there are several unused imports (shutil, subprocess and locale). Maybe worth stripping them out?